pages/people.md

```diff
@@ -29,13 +29,14 @@ based on commits.
 | [Eric Newton](https://github.com/ericnewton)             |                                     | [ET][et]   |
 | [Garvit Bansal](https://github.com/Garvit244)            | [Flipkart](https://www.flipkart.com)| [IST][ist] |
 | [Michael Wall](https://github.com/mjwall)                |                                     | [ET][et]   |
-| [Ravikrishna Adepu](https://github.com/adepuravikrishna) | [SAP Labs LLC](https://sap.com)     | [PDT][pdt] |
+| [Ravikrishna Adepu](https://github.com/adepuravikrishna) | [SAP Labs LLC](https://sap.com)     | [PT][pt] |
+| [Kenneth McFarland](https://github.com/kpm1985)          |                                     | [PT][pt] |
 
 [tc]: http://www.tetraconcepts.com/
 [hw]: https://hortonworks.com/
 [pt]: http://www.ptech-llc.com/
 [bah]: https://www.boozallen.com/
 [et]: https://www.timeanddate.com/time/zones/et
-[pdt]: https://www.timeanddate.com/time/zones/pdt
+[pt]: https://www.timeanddate.com/time/zones/pt
```

Comment:
I took a quick look at this last night and didn't see this problem but when I tried to rebuild the website this morning I noticed the Peterson Technologies links were pointing to time and date.   Then I saw `pt` was reused, funny.  We should probably use slightly longer link ref names.  I fixed this in #94.